aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -354,38 +428,38 @@ The following table contains a set of common mistakes found in messaging, along
 | as it has                                                              | because it has                                                          | In general, do not use "as" where you can use "because"                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | make sure                                                              | ensure                                                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-| check if                                                               | ensure that                                                             | Note: "check if" or "verify that" are never valid resolutions because they don't tell you what to do or how to do it. A resolution must be actionable. For example: "Check if the required CA certificates have been added and are valid" is not a good resolution. It does not tell you what to do if the certs have not been added or is the certs have been added but are not valid. Our messaging must lead the user to success -- not leave them guessing.                                |
+| check if                                                               | ensure that                                                             | Note: "check if" or "verify that" are never valid resolutions because they don't tell you what to do or how to do it. A resolution must be actionable. For example: "Check if the required CA certificates have been added and are valid" is not a good resolution. It does not tell you what to do if the certs have not been added or if the certs have been added but are not valid. Our messaging must lead the user to success -- not leave them guessing.                                |
```

Comment:
```suggestion
| check if                                                               | ensure that                                                             | Note: "check if" or "verify that" are never valid resolutions because they don't tell you what to do or how to do it. A resolution must be actionable. For example: "Check if the required CA certificates have been added and are valid" is not a good resolution. It does not tell you what to do if the certs have not been added or if the certs have been added but are not valid. Our messaging must lead the user to success &mdash; not leave them guessing.                                |
```